FAQ.md

```diff
@@ -14,7 +14,7 @@ In addition, APISIX has dynamic routing and hot loading of plug-ins, which is es
 
 ## What's the performance of APISIX?
 
-One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md。
+One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md.
```

Comment:
It is recommended to remove the symbol here, keep the same as Chinese version.

Reply:
Also, we'd better use the `[Title](Link)` Markdown Hyperlink syntax here.

Reply:
fixed. I have change the Hyperlink format.